### ISSUE TITLE
No information regarding installation of the extension

### ISSUE THREAD

**Issue by rohanprabhu** [intoli/remote-browser#74]
The README makes multiple references to a 'Remote Browser' extension, but neither provides a link to it or any steps to to install it from source (possible the src/extension directory). I am really not sure if I am missing something, but it feels like a lot of people might get stuck here when trying to run remote-browser against their daily use browser.

**Comment by sangaline** [comment]
Thanks, that's a good point. We should really add instructions and publish the extension in the Chrome Web Store and Firefox Addon directory. In the meantime, the location of the extension that you need to install is in `node_modules/remote-browser/extension/` if you've installed the `remote-browser` module using npm. If you're working from the git repo, it will be in `dist/extension/` after you run `yarn build`. The source code for the extension is located in `src/extension/` as you mentioned, but the build step is required before it can be loaded into a browser.

To install the extension temporarily, you can follow the appropriate instructions for your browser:

- [Firefox temporary addon instructions](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Temporary_Installation_in_Firefox)
- [Chrome "load an unpacked extension" instructions](https://developer.chrome.com/extensions/getstarted#manifest)

After that, the animated GIF in the README should mostly illustrate how to connect. You basically run something like this.

```bash
const { Browser } = require('remote-browser');
const browser = new Browser();
browser.listen().then(port => console.log('Listening on port', port));
```

It will echo out a port that you need to enter in the interface for the extension (they should both default to 8000).

Please let me know if you run into any problems!

**Comment by rohanprabhu** [comment]
Thank you so much for the prompt reply. I tried doing the same (I've tried both methods, using the git repo and from my node_modules directory), but I'm getting a whole bunch of errors in the Firefox panel. The output that I see is this: https://gist.github.com/rohanprabhu/6a997923cad6f8384b5b18dea791c39b

I am running **Firefox 61.0.1** on ArchLinux and if this helps:

    ~ ツ uname -a
    Linux rohanarch.local 4.17.5-1-ARCH #1 SMP PREEMPT Sun Jul 8 17:27:31 UTC 2018 x86_64 GNU/Linux

It seems to work in Chrome, but I get the following warnings. This might be relevant to you and your team:

![image](https://user-images.githubusercontent.com/136263/42894151-c6d16776-8ad3-11e8-9a91-97f5f12a90f0.png)

I am running **Chrome 67.0.3396.99**. The extension is working on Chrome. I am able to run whatever is shown in the video walkthrough for running it against a local browser. Please let me know if I can be of any more help.

**Comment by sangaline** [comment]
Yeah, the permissions list needs to be cleaned up to fix those warnings. Are you certain that the extension doesn't run when you load it in Firefox? Those manifest errors are expected, but it should still work.

**Comment by rohanprabhu** [comment]
Apologize for the late reply. I can confirm that the remote connection button does not show up in the toolbar. Let me know if I'm doing something wrong: https://imgur.com/a/hFBsLd4
